acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -0,0 +1,39 @@
+require 'puppet/acceptance/common_utils'
```

Comment:
If this file is included; then we should completely replace acceptance/setup/aio/010_Install.rb
There's some code at the bottom of acceptance/setup/aio/010_Install.rb that needs retained. It can perhaps move to setup/common/020_Configure_Pxp_Agents.rb ?